modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -153,13 +153,17 @@
 
       {{/vendorExtensions.x-streaming}}
       if (local_var_response$status_code >= 200 && local_var_response$status_code <= 299) {
-        local_var_response$content
+        if (is.raw(local_var_response$content)) {
+          return(local_var_response)
+        } else {
+          return(local_var_response$content)
+        }
       } else if (local_var_response$status_code >= 300 && local_var_response$status_code <= 399) {
-        local_var_response
+        return(local_var_response)
```

Comment:
I can revert the line 157 change to return the raw response if you prefer, but it seemed consistent and more helpful with response codes not in the 200s to return the API response object instead of a raw vector.

Reply:
would prefer remove this change from this PR and file another PR for this breaking change.

Reply:
Done, see #20142